app/src/main/java/com/github/se/studybuddies/ui/shared_elements/SharedUI.kt

```diff
@@ -365,6 +339,29 @@ private fun MenuButton(onClick: () -> Unit) {
       }
 }
 
+@Composable
+fun SaveButton(enabled: Boolean, save: () -> Unit) {
+  Button(
+      onClick = save,
+      enabled = enabled,
+      modifier =
+          Modifier.padding(0.dp)
+              .width(300.dp)
+              .height(50.dp)
+              .background(color = Color.Transparent, shape = RoundedCornerShape(size = 10.dp))
+              .testTag("save_button"),
+      colors =
+          ButtonDefaults.buttonColors(
+              containerColor = Blue,
+          )) {
+        Text(
+            stringResource(R.string.save),
+            color = White,
+            modifier = Modifier.testTag("save_button_text"))
+      }
+}
+
+/*
 @Composable
```

Comment:
The chunk of commented out code could probably be deleted

Reply:
Done!